twemproxy/datadog_checks/twemproxy/twemproxy.py

```diff
@@ -8,44 +8,30 @@
 
 from datadog_checks.base import AgentCheck, ensure_unicode
 
+GLOBAL_STATS = set(['curr_connections'])
 
-GLOBAL_STATS = set([
-    'curr_connections',
-])
-
-GLOBAL_STATS_RATES = set([
-    'total_connections'
-])
-
-POOL_STATS = set([
-    'client_connections',
-    'server_ejects',
-])
-
-POOL_STATS_RATES = set([
-    'client_eof',
-    'client_err',
-    'forward_error',
-    'fragments'
-])
-
-SERVER_STATS = set([
-    'server_connections',
-    'server_timedout',
-])
-
-SERVER_STATS_RATES = set([
-    'in_queue',
-    'out_queue',
-    'in_queue_bytes',
-    'out_queue_bytes',
-    'requests',
-    'request_bytes',
-    'responses',
-    'response_bytes',
-    'server_err',
-    'server_eof'
-])
+GLOBAL_STATS_RATES = set(['total_connections'])
```

Comment:
Those would be good candidates to use the {} set syntax.

Reply:
done!